pages/common/unlink.md

```diff
@@ -0,0 +1,7 @@
+# unlink
+
+> Remove a file from a filesystem.
```

Comment:
Let us change this to "Delete a name from the file system" since this is more precise. It does not always delete a file.

Reply:
How about:
`Decrement the link count of a file, deleting the file if the link count reduces to '0'`
This is in view of the change suggested by @agnivade 
I wanted to use the word `remove` instead of `deleting` but am unable to decide...